rhel.Dockerfile

```diff
@@ -9,26 +9,28 @@
 #   Red Hat, Inc. - initial API and implementation
 #
 
-# UPSTREAM: use devtools/go-toolset-rhel7 image so we're not required to authenticate with registry.redhat.io
-# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/go-toolset
-FROM registry.access.redhat.com/devtools/go-toolset-rhel7:1.11.13-11.1571302666 as builder
-ENV PATH=/opt/rh/go-toolset-1.11/root/usr/bin:$PATH
-
-# DOWNSTREAM: use rhel8/go-toolset; no path modification needed
-# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/go-toolset
-# FROM registry.redhat.io/rhel8/go-toolset:1.12.8-18 as builder
```

Comment:
cc: @nickboldt Do you need this line?

Reply:
yes -- we don't have a system in place to swap from go toolset rhel7 to rhel8 go toolset on the fly, and since there's some resistance in upstream to rely on base images which require authentication from the registry, we're stuck for now having rhel7 in upstream and using rhel8 in downstream (to avoid CVP failures when the content sets get confused by a build which relies on BOTH rhel7 and rhel8 repos)

Reply:
Yes but is the commented line necessary or can the modification be done on the existing `FROM`?

Reply:
Commented URLs are used to drive automated updates via https://github.com/redhat-developer/codeready-workspaces/blob/master/product/updateBaseImages.sh

but the comments about up vs. down ... you could remove those (but they do clarify why we list two FROMs).

Reply:
@nickboldt  can you clarify wich exactly lines I can remove?

Reply:
lines like
```
# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhel8/go-toolset
``` 
which are located one line before a matching `FROM` are used for keeping base images current.

The rest are for information purposes only. But if you remove them people will ask "why are there two different FROM lines and why is one commented out?"

So... unless there's a performance reason why these comments are hindering the build time I'd say just leave them alone.

Reply:
> Yes but is the commented line necessary or can the modification be done on the existing FROM?

We COULD set up an on-the-fly replacement sed transform to replace rhel7 go toolset with rhel8 one. 

But leaving the commented code in place makes it more clear that we HAVE to do that instead of buying the logic in a Jenkinsfile downstream in CRW repo.

Reply:
So TL;DR, please leave this alone as is. Downstream needs it and it doesn't hurt upstream.

Reply:
@nickboldt I would like to have as much as possible similarity with https://github.com/eclipse/che-plugin-broker/blob/master/build/artifacts/rhel.Dockerfile. If you don't mind can we decide whatever to remove these lines altogether in the separate issue?

Reply:
agreed, but then you'll need to undo the change here to keep them